.github/workflows/checks.yml

```
@@ -83,6 +83,11 @@ jobs:
           asdf install starknet-foundry 0.6.0
           asdf global starknet-foundry 0.6.0
 
+      - name: Check Cairo contracts formatting
+        run: |
+          cd ${{ github.workspace }}/Tests/StarknetTests/Resources/Contracts/src
+          scarb fmt --check
+
```

as a nitpick, as in this scenario it's fine, but in bash scripts try to use `pushd` and `popd` instead of `cd` :) That way you do not have to hold an info about previous cwd (and do not have to use something like `cd -`, which can be unreliable)

If you wanted to improve in this particular case though, instead of cd you could use working directory directive, which would be a lot cleaner:

```suggestion
      - name: Check Cairo contracts formatting
        working-directory: ./Tests/StarknetTests/Resources/Contracts/src
        run: scarb fmt --check

```

---

Sure, thanks for suggestions.